packages/terra-arrange/src/terra-dev-site/doc/common/examplesetup.scss

```diff
@@ -0,0 +1,23 @@
+// stylelint-disable scss/at-mixin-pattern
+
+:local {
+  @mixin align-example-common() {
+    background-color: #000;
+    display: block;
+    height: 20px;
+    width: 40px;
+  }
+
+  .text-with-blue-border {
```

Comment:
nit: Let's change this to `outlined-placeholder`

Reply:
Fixed here https://github.com/cerner/terra-core/pull/2427/commits/1b2f4166cbfec7f0bcf85035c64e83ec7746a85f.